all.json

```diff
@@ -63,6 +63,7 @@
     "auth-metamask.com",
     "authenticate-wallet.com",
     "authenticatewalletlive.com",
+    "axielnfinity.info",
```

Comment:
Not 100% sure about this one - cannot actually properly evaluate it since it is non-Polkadot ecosystem related.

Reply:
COrrect, my bad, i checked and It is not related to polkadot. The legitimate site is axieinfinity.com and not .info

Reply:
I generally don't mind these extra ones (as long as they don't flood and step on the toes of other scam lists), but in this case it is really not obvious, so just cannot 100% validate if it above-board or not. (Not that I don't trust the listing, just that in general the scammer sites are getting quite good and here just don't have a frame of reference)